src/misc/constants.ts

```diff
@@ -36,6 +36,26 @@ export const SOUND_UPDATE = 16;
 
 export const MAX_LAYER : number = 32;
 
+
+///All relevant planes used by Yogstation categorized into an enum of their values
+export const enum Planes{
+    LOWEST_EVER_PLANE = -100,
+    CLICKCATCHER_PLANE = -99,
+    SPACE_PLANE = -95,
+    FLOOR_PLANE = -2,
+    GAME_PLANE = -1,
+    BLACKNESS_PLANE = 0,
+    EMISSIVE_BLOCKER_PLANE = 12,
+    EMISSIVE_PLANE = 13,
+    EMISSIVE_UNBLOCKABLE_PLANE = 14,
+    LIGHTING_PLANE = 15,
+    O_LIGHTING_VISUAL_PLANE = 16,
+    FLOOR_OPENSPACE_PLANE = 17,
+    BYOND_LIGHTING_PLANE = 18,
+    CAMERA_STATIC_PLANE = 19,
+    HIGHEST_EVER_PLANE = 110,

```

Comment:
I didn't pick -10000 to 10000 out of thin air, that's just what the range of acceptable plane values are:

![image](https://github.com/yogstation13/demo-viewer/assets/3681297/86152472-bfbb-411d-8a1a-30f07e39416f)

don't change the range for no reason.

Reply:
sure, i just set it to match the exact defines yog was using so people would be familiar with constants as an analog for the defines from the get go but i'll change it back

Reply:
HIGHEST_EVER_PLANE is not a define in yogs though?